src/Schema/Schema.php

```diff
@@ -179,7 +184,7 @@ public function getDirectives(): array
     /**
      * @return array
      */
-    public function getTypeMap(): array
+    public function getTypeMap(): iterable
```

Comment:
Any particular reason why you chose iterable over array in the contracts?

Reply:
`interable` indicated in the interface and I was afraid that in 7.1 this will not work, because redefinition is only available from 7.4. 

However, later, when I manually checked it, I was convinced that even in php 7.1 this code works: http://sandbox.onlinephpfunctions.com/code/e19dffa7e679c2a9fbaff1592e047eae4c07b19b So I returned everything back. Perhaps in this place I just forgot =)

Reply:
Well, if you want to keep `iterable` in the contracts then I guess we'll have to use `iterable` here too.